test/classes/Gis/GisVisualizationTest.php

```diff
@@ -4,28 +4,31 @@
 
 namespace PhpMyAdmin\Tests\Gis;
 
+use PhpMyAdmin\DatabaseInterface;
 use PhpMyAdmin\Gis\GisVisualization;
 use PhpMyAdmin\Tests\AbstractTestCase;
 
 /** @covers \PhpMyAdmin\Gis\GisVisualization */
 class GisVisualizationTest extends AbstractTestCase
 {
+    private DatabaseInterface|null $dbi = null;
```

Comment:
Don't need `null` here. The `setUp()` method is treated as a constructor.

```suggestion
    private DatabaseInterface $dbi;
```